### DISCUSSION TITLE
how about get filesystem cache use the hook while start the filesystem?

### DISCUSSION THREAD

**Discussion by Idletalk** [webpack/webpack#12788] [Q&A] [answered]
hi, i have a question about: i want to get the cache file when start filesystem. i try to use the hook about "done" like this
compiler.hooks.done.tap('', () => {
// get the cache filesystem to do something
})
but i field.
which hook in compiler or compiler.cache can i use to get the filesystem cache?

**Comment by alexander-akait** [comment, marked as answer]
Why you need cache file?

**Reply by Idletalk** [reply]
Because i want to reuse the cache at another place, so i want to store the cache file to another palce when the cache file output

**Reply by alexander-akait** [reply]
It is impossible, because we store absolute paths in cache, you need this https://github.com/webpack/webpack/issues/10400 to support it

**Reply by Idletalk** [reply]
i see, thank you for you help
